app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-feature android:name="android.hardware.camera.any" />
+
+    <uses-permission android:name="android.permission.CAMERA" /> <!-- Devices running Android 12L (API level 32) or lower -->
+    <uses-permission
+        android:name="android.permission.READ_EXTERNAL_STORAGE"
```

Comment:
It seems we do not (or not yet?) currently use external file/cache access APIs in this project.
Is this `android.permission.READ_EXTERNAL_STORAGE` permission really needed then?

Reply:
You are right. It should be removed.